izhi.py

```diff
@@ -48,7 +48,7 @@ def createcell(section, C, k, vr, vt, vpeak, a, b, c, d, celltype, cellid):
     cell.d = d
     cell.celltype = celltype # Set cell celltype (used for setting celltype-specific dynamics)
     cell.cellid = cellid # Cell ID for keeping track which cell this is
-    cell.t0 = .0
+    #cell.t0 = .0
```

Comment:
forgot why this is commented out

Reply:
if I do not comment it out I get the following error:
```
python main.py

Setting parameters...
Benchmarking...
  Running at 114% default speed (114% total)

Creating simulation of 8512 cells for 1.0 s on 1 hosts...
Traceback (most recent call last):
  File "/gpfs/bbp.cscs.ch/data/project/proj16/kotsalos/HACKRON2022/nrn-modeldb-ci/test/Dura-BernalEtAl2017/main.py", line 40, in <module>
    network.runTrainTest2targets()
  File "/gpfs/bbp.cscs.ch/data/project/proj16/kotsalos/HACKRON2022/nrn-modeldb-ci/test/Dura-BernalEtAl2017/network.py", line 78, in runTrainTest2targets
    createNetwork()
  File "/gpfs/bbp.cscs.ch/data/project/proj16/kotsalos/HACKRON2022/nrn-modeldb-ci/test/Dura-BernalEtAl2017/network.py", line 333, in createNetwork
    cell = celltypes[gid](s.dummies[s.cellsperhost], cellid=gid) # Create a new cell of the appropriate type (celltypes[gid]) and store it
  File "/gpfs/bbp.cscs.ch/data/project/proj16/kotsalos/HACKRON2022/nrn-modeldb-ci/test/Dura-BernalEtAl2017/izhi.py", line 57, in RS
    cell = createcell(section, C, k, vr, vt, vpeak, a, b, c, d, celltype, cellid)
  File "/gpfs/bbp.cscs.ch/data/project/proj16/kotsalos/HACKRON2022/nrn-modeldb-ci/test/Dura-BernalEtAl2017/izhi.py", line 51, in createcell
    cell.t0 = .0
LookupError: 't0' is not a defined hoc variable name.
```

Reply:
Ok will check on my side

Reply:
So `t0` from `izhi2007.mod` is ASSIGNED only therefore it's used for internal calculations:

```bash
./x86_64/special
NEURON -- VERSION 8.2.0+ HEAD (156b9dee+) 2022-07-01
Duke, Yale, and the BlueBrain Project -- Copyright 1984-2021
See http://neuron.yale.edu/neuron/credits

Additional mechanisms from files
 "izhi2007.mod" "nsloc.mod" "stdp.mod" "vecevent.mod"
oc>objref c
oc>c = new Izhi2007()
oc>c.t0 = 1
t0 not a public member of Izhi2007
./x86_64/special: Izhi2007 t0
 near line 4
 c.t0 = 1
         ^
oc>
```
Most probably older NEURON versions allowed accessing ASSIGNED variables. So there are two solutions:

- remove `cell.t0 = 0`
- add `t0` to the `NEURON` block in `izhi2007.mod` (I'm leaning more towards this solution)

However not sure of the implications wrt the model itself. @ramcdougal @nrnhines any idea?